server/gitrest/src/externalStorageManager.ts

```diff
@@ -0,0 +1,61 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import Axios from "axios";
+import * as winston from "winston";
+
+export interface IExternalStorageManager
+{
+    readAndSync(tenantId: string, documentId: string): Promise<boolean>;
+
+    writeFile(tenantId: string, ref: string, sha: string, update: boolean): Promise<void>;
+}
+
+/**
+ * Manages api calls to external storage
+ */
+export class ExternalStorageManager implements IExternalStorageManager {
+    constructor(private endpoint: string) {
+    }
+
+    public async readAndSync(tenantId: string, documentId: string): Promise<boolean> {
+        if (process.env.external_storage_enabled != "true") {
+            winston.info("External storage is not enabled");
+            return false;
+        }
```

Comment:
You dont need to check precess.env anymore. You can just check the config directly.